server/methods/accounts/accounts.js

```diff
@@ -678,17 +678,31 @@ export function inviteShopMember(options) {
   const currentUser = Meteor.users.findOne(this.userId);
   const currentUserName = getCurrentUserName(currentUser);
   const emailLogo = getEmailLogo(primaryShop);
-  const token = Random.id();
   const user = Meteor.users.findOne({ "emails.address": email });
+  let token = Random.id();
   let dataForEmail;
   let userId;
+  let tpl;
+  let subject;
 
+  // If the user already has an account, send informative email, not "invite" email
   if (user) {
-    userId = user._id; // since user exists, we promote the account
+    // The user already exists, we promote the account, rather than creating a new one
+    userId = user._id;
     Meteor.call("group/addUser", userId, groupId);
+
+    // do not send token, as no password reset is needed
+    token = null;
```

Comment:
@spencern when reviewing, please take a look at the way I did the token / url here, not sure if we need both of these.

Initially, I just set the token to be null here, which would in turn return just the `Meteor.absoluteUrl()` in the `getEmailUrl` function on line 1017.

After doing this, I decided to add the extra url `const` here, in case we would ever want to pass another URL directly in. I think this was is better, however I wasn't sure about removing the update to the `token` here, because it doest make sense to leave it in case another URL isn't provided.